.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,26 @@
-## Pull Request Guidelines
+## Checklist for submitting a Pull Request (delete this when complete)
```

Comment:
"delete this when complete" is not what we want here. 
If people use this template to add a more detailed release note, at least that block needs to remain.  
In fact, that is the primary reason for this PR template so if nothing else, that should be highlighted.